proto/README.md

```diff
@@ -0,0 +1,37 @@
+# Miden proto build
+
+This crate contains the proto definitions for the gRPC endpoints at `proto/proto`.
+
+Also exposes the compiled proto `FileDescriptorSet` to generate the Rust bindings. See next how to use it.
+
+# Usage
+Likely, you will need to set up a `build.rs` file that generates the proto bindings at build time. How use the proto builder:
+
+```rust
+use std::{fs, path::PathBuf};
+use miden_node_proto_build::rpc_file_descriptor;
+
+fn main() {
+    let dst_dir = PathBuf::from("./generated");
```

Comment:
This will go to `src/generated`, right? If so, I'd add a simple comment explaining this.